doc/docs/master/enterprise/auth/authorization/index.md

```diff
@@ -42,54 +42,42 @@ Pachyderm has a number of predefined roles granting permissions to its Resources
 Those Roles are listed here in no specific hierarchical order. 
 Some might inherit a set of permissions from another.
 
-- **repoReader**: Can consume data from a repo, but cannot edit them.
+### Repo Roles
```

Comment:
I thought it would make sense to group the different types of roles under different headings, depending on what resources they can be applied to?